CodeEditModules/Modules/StatusBar/src/StatusBarItems/StatusBarDrawer.swift

```diff
@@ -12,14 +12,40 @@ internal struct StatusBarDrawer: View {
     @ObservedObject
     private var model: StatusBarModel
 
+    @State
+    private var searchText = ""
+
     internal init(model: StatusBarModel) {
         self.model = model
     }
 
+    var height: CGFloat {
+        if model.isMaximized {
+            return model.maxHeight
+        }
+        if model.isExpanded {
+            return model.currentHeight
+        }
+        return 0
+    }
+
     internal var body: some View {
         TerminalEmulatorView(url: model.workspaceURL)
             .frame(minHeight: 0,
-                   idealHeight: model.isExpanded ? model.currentHeight : 0,
-                   maxHeight: model.isExpanded ? model.currentHeight : 0)
+                   idealHeight: height,
+                   maxHeight: height)
```

Comment:
I guess the problem lies within this change

Reply:
Ok just tried to revert this but it doesn't fix the problem. On current `main` the bug doesn't occur.

Reply:
reverting this doesn't change it. However swapping out the `safeAreaInset` for a `ZStack` does the trick 👍

Reply:
Nice! Once the lint errors are resolved I'll merge this!